AUTHORS

```diff
@@ -0,0 +1,28 @@
+# This file lists all individuals having contributed content to the repository.
```

Comment:
I'm personally not a fan of adding a file like this, because one could simply run a git command to get this information. (shortlog -s). It gets stale quickly and adds an additional maintenance burden (or artifacts in the git changelog just to update this file).

I'm curious how others feel though.

Reply:
> being able to get this information by simply running a git command is an argument against having it at all, regardless of how it is generated

agreed. for now let's avoid the complexity.